MapboxNavigation/Style.swift

```diff
@@ -512,3 +514,51 @@ class ManeuverContainerView: UIView {
         }
     }
 }
+
+/// :nodoc:
+@objc(MBStatusView)
+class StatusView: UIView {
+    @IBOutlet weak var activityIndicatorView: UIActivityIndicatorView!
+    @IBOutlet weak var textLabel: UILabel!
+    @IBOutlet weak var topConstraint: NSLayoutConstraint!
+    
+    func show(_ title: String, showSpinner: Bool, duration: TimeInterval) {
+        textLabel.text = title
+        activityIndicatorView.isHidden = !showSpinner
+        activityIndicatorView.startAnimating()
+        
+        show()
```

Comment:
Maybe call these `show()` function something a little different? hard to know what they each do.